toolchain/check/return.cpp

```diff
@@ -0,0 +1,175 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/check/context.h"
+#include "toolchain/check/convert.h"
+#include "toolchain/parse/tree.h"
+#include "toolchain/sem_ir/inst.h"
+
+namespace Carbon::Check {
+
+static auto GetCurrentFunction(Context& context) -> SemIR::Function& {
```

Comment:
Would it be worth quick comments on the `static` functions in this file?

Reply:
Done.